src/js/tabs/motors.js

```diff
@@ -365,8 +365,35 @@ TABS.motors.initialize = function (callback) {
         // select current mixer configuration
         mixerListElement.val(FC.MIXER_CONFIG.mixer).change();
 
+        function validateMixerOutputs() {
+            MSP.promise(MSPCodes.MSP_MOTOR).then(() => {
+                const mixer = FC.MIXER_CONFIG.mixer;
+                const motors = mixerList[mixer - 1].motors;
+                // initialize for models with zero motors
+                self.numberOfValidOutputs = motors;
+
+                for (let i = 0; i < FC.MOTOR_DATA.length; i++) {
+                    if (FC.MOTOR_DATA[i] === 0) {
+                        self.numberOfValidOutputs = i;
+                        // filter out models with zero motors defined.
```

Comment:
:-) This comment is now outdated.

Reply:
Put it there to explain the second condition.

Reply:
Fixed ;-)